packages/compass-components/src/components/document-list/element.tsx

```diff
@@ -428,6 +428,7 @@ export const HadronElement: React.FunctionComponent<{
   lineNumberSize: number;
   onAddElement(el: HadronElementType): void;
   extraGutterWidth?: number;
+  onAddToQuery: (field: string, value: any) => void;
```

Comment:
This is probably not meaningful in all contexts, so should be optional (and the context menu item only available if it is set)?

Reply:
ah yeah, that was my original thinking